scarlet/observation.py

```diff
@@ -188,6 +315,31 @@ def get_LowRes(self):
                                  wcs=self.frame.wcs,
                                  channels=self.frame.channels)
 
+    def _project(self, frame, images=None):
```

Comment:
that method has another signature as previous ones, I suggest to use this one here for all of them:
`project(self, frame, model=None)` or `..., image=None)`.
Still not a fan of "project"...

Reply:
I used `images` instead of `model` because unlike `Component._project`, which projects the model into a `Frame`, this projects an image (by default `observation.images`) into a `Frame`, where `images` must be located in `observation.frame`. Originally I didn't have the `images` argument at all but needed to use this to project `observation.weights` as well as `observation.images`.

Reply:
I was referring to the order of the arguments. `frame` is required in all cases, but the secondary argument (either `model` or `images`) is not. This is why I'd prefer this signature here of the earlier incarnations.

Reply:
Ah, I see. Yes, I agree that your suggestion is better.